lib/shopify_api/errors/log_level_not_found_error.rb

```diff
@@ -0,0 +1,9 @@
+# typed: strict
+# frozen_string_literal: true
+
+module ShopifyAPI
+  module Errors
+    class LogLevelNotFoundError < StandardError
```

Comment:
I wonder if we should be raising an error if log level is misconfigured? Do we want to potentially break an API call or server instance because of a bad config? I'd think we'd want to just send a warning to logger but keep the app running. I'd hate to bring down someone's app because they couldn't tell they misconfigured something until it started processing requests

Reply:
Perhaps only raise in dev/test environments?

Reply:
Is there a dev/test environment in ruby? Or is that just relevant for rails? I did a quick google and could only find things for rails. So I am just sending a log: 
```ruby
      sig { params(log_level: Symbol).returns(T::Boolean) }
      def valid_log_level?(log_level)
        return true if ::ShopifyAPI::Logger::LOG_LEVELS.keys.include?(log_level)

        ShopifyAPI::Logger.warn("#{log_level} is not a valid log_level. "\
          "Valid options are #{::ShopifyAPI::Logger::LOG_LEVELS.keys.join(", ")}")

        false
      end
```
Since the log level isn't set it may have to rely on the previous `setup` call or our default inside the logger which is `info`

Reply:
> Is there a dev/test environment in ruby? 

Ah, no 🤦